vue-components/src/components/DateInput.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-DateInput' ]"
+	>
+		<InputWithExtender
+			:label="label"
+			:value="value"
+			:error="error"
+			:feedback-type="feedbackType"
```

Comment:
InputWithExtender doesn't have a prop "feedback" type. The error prop is good enough

Reply:
oh nice. Thanks. done now.